skypy/neutral_hydrogen/__init__.py

```diff
@@ -1,4 +1,16 @@
-'''This module contains methods that model the properties of neutral hydrogen.
-'''
+"""
+This module contains methods that model the properties of neutral hydrogen
+emission in the Universe.
 
+Merger Rates
+================
 
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+
+   pr_halo_model
```

Comment:
I don't think this is the best name for the model, mostly because it makes me think that this is a halo model. Suggestions would be `hi_mass_pr17` or `padmanabhan_refregier_17` (which not everyone likes).